readme/Compile.md

```diff
@@ -0,0 +1,53 @@
+# Compilation and Testing
+
+This file contains platform-independent instructions for compiling and developing locally. For instructions on how to create platform-specific packages, or instructions on preliminary steps for building Xournal++, please see these platform-specific instructions:
+
+- [LinuxBuild.md](./LinuxBuild.md)
+- [MacBuild.md](./MacBuild.md)
+- [WindowsBuild.md](./WindowsBuild.md)
+
+## Get sources
+
+```sh
+git clone http://github.com/xournalpp/xournalpp
+cd xournalpp
+```
+
+## Compile
+
+```sh
+mkdir build
+cd build
+
+cmake ..
+cmake --build . # For a faster build, set the flag -DCMAKE_BUILD_TYPE=RelWithDebInfo
+```
+
+- Use `cmake-gui ..` to graphically configure CMake
+- With Cairo 1.16 PDF Bookmarks will be possible, but this version is not yet
+commonly available; therefore the Cairo PDF Export does not include PDF Bookmarks
```

Comment:
This line should be removed. PDF bookmarks are supported when compiling with Cairo >= 1.16 (we have a macro to check the version).